README.md

```diff
@@ -1 +1,65 @@
-# Seminar--Capstone-
\ No newline at end of file
+# Seminar (Capstone Project)
+
+> This project contains home and about pages of a behance design template to imitate a prototype of annual seminar on trading.
+> The features in this project were created using javascript, html and css.
+>Javascript features (object) is also inculcated.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo Links
+
+[Live Demo Link](https://yussufob.github.io/Seminar--Capstone-/)
+[Live Demo Link](https://www.loom.com/share/b3fbdb57749f41e786f97740d492b5cc)
```

Comment:
## Optional
- [ ] I suggest that you show that one link is a live demo then another one is a video demo to be clear you can simply do:

## Live demo
---

## Video demo
---